### PR TITLE
Added my UT47.2 Swedish layout with tap dance function.

### PR DESCRIPTION
**Pull request by hvpkod**
## Description

Keymap to be used with Eurkey for easy access to Swedish characters on the #first layer.

Links:

- https://eurkey.steffen.bruentjen.eu/

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* NA

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ut472/keymaps/hvp/config.h [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+#define TAPPING_TERM 150
+#define PERMISSIVE_HOLD
+#define IGNORE_MOD_TAP_INTERRUPT
+
+#include "config_common.h"
```
Comment (drashna) [inline keyboards/ut472/keymaps/hvp/config.h]: This line is not actually needed, and should be removed. 

```suggestion
```

File: keyboards/ut472/keymaps/hvp/keymap.c [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+  RESET, _______, _______, _______, KC_MUTE,     KC_MPLY,      KC_MSTP, KC_MPRV, KC_VOLD, KC_VOLU, KC_MNXT
+),
+};
+
+// Tap dance 1
+void dance_1_finished(qk_tap_dance_state_t *state, void *user_data)
+{
+  if (state->count == 2)
+  {
+    register_code(KC_SCLN);
```
Comment (drashna) [inline keyboards/ut472/keymaps/hvp/keymap.c]: A lot of these could be replaced with `tap_code`, `register_code16(RALT(kc))`, etc.

**Comment by noroadsleft** [comment]
Looks like https://github.com/qmk/qmk_firmware/pull/6861/commits/b81a1ae36a564e6296cc19f23ff1782222cbb5ab broke your keymap on our side. I think you need to commit your userspace changes to this branch and push it up.

**Comment by hvpkod** [comment]
> Looks like [b81a1ae](https://github.com/qmk/qmk_firmware/commit/b81a1ae36a564e6296cc19f23ff1782222cbb5ab) broke your keymap on our side. I think you need to commit your userspace changes to this branch and push it up.

ts committed in branch 6857.

File: keyboards/ut472/keymaps/hvp/keymap.c [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+#include "hvp.c"
```
Comment (drashna) [inline keyboards/ut472/keymaps/hvp/keymap.c]: This file doesn't seem to exist?

Comment (drashna) [reply keyboards/ut472/keymaps/hvp/keymap.c]: n/m

**Pull request merged**